File: Ubuntu (Code Blocks)/Base Converter.py
from math import *

def convert(fN,fB,sB):
    try:
        tVT=int(str(fN),fB)
        sN=""
        while tVT:
            sN+=str(tVT%sB)
            tVT//=sB
        return(sN[::-1])
    except ValueError:
        return("Error 003: Invalid input detected/")

#convert(int(input("What is the number?\n")),
#          int(input("What is the current base? (1 < x < 11)\n")),
#          int(input("What is the desired base? (1 < x < 11)\n")))
#11110100001000111111 is 999999 in base 10

File: C++/Base Converter.cpp
#include <stdio.h>
#include <iostream>
#include <cmath>
#include <string>
using namespace std;

int main()
{
	cout << "What is the number?\n";				string fN;	cin >> fN;
	cout << "What is the current base? (<=36)\n";	int fB;		cin >> fB;
	cout << "What is the desired base? (<=10)\n";	int sB;		cin >> sB;
	if (1<fB<37 && 1<sB<11)
	{
		long tVT = stoi(fN, nullptr, fB);
		string sN;
		while (tVT)
		{
			sN += to_string(tVT%sB);
			tVT = floor(tVT / sB);
		}
		cout << "\nYour number is ";
		for (string::reverse_iterator rit = sN.rbegin(); rit != sN.rend(); ++rit)
			cout << *rit;
		cout << ".\n";
	}
	else
	{
		cout << "\nStop wasting time.\n";
	}
}
